src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -1,30 +1,85 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Table from 'paragon/src/Table';
+import Button from 'paragon/src/Button';
 
-const AssetsTable = ({ assetsList }) => (
-  (!assetsList.length) ? (
-    <span>Loading....</span>
-  ) : (
-    <Table
-      columns={[
-        {
-          label: 'Name',
-          key: 'display_name',
-        },
-        {
-          label: 'Type',
-          key: 'content_type',
-        },
-        {
-          label: 'Date Added',
-          key: 'date_added',
-        },
-      ]}
-      data={assetsList}
-    />
-  )
-);
+import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
+
+class AssetsTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      finalAssetsList: [],
```

Comment:
Just a nit: I might name this something like `displayAssetsList` -- it's not exactly *final* since we update it when the underlying data changes ` -- but up to you.